### ISSUE TITLE
python finetune_pruned.py --weights runs/test/exp16/pruned_model.pt --adam --epochs 100 --cfg models/yolov5m.yaml --data data/struct.yaml --device 0,1,2,3  一直卡着

### ISSUE THREAD

**Issue by RomanticWithoutStatus** [midasklr/yolov5prune#109]
detect input :  ['model.0.conv.bn', 'model.1.bn', 'model.2.cv3.bn', 'model.3.bn', 'model.4.cv3.bn', 'model.5.bn', 'model.6.cv3.bn', 'model.7.bn', 'model.8.cv2.bn', 'model.9.cv3.bn', 'model.10.bn', 'model.10.bn', ['model.10.bn', 'model.6.cv3.bn'], 'model.13.cv3.bn', 'model.14.bn', 'model.14.bn', ['model.14.bn', 'model.4.cv3.bn'], 'model.17.cv3.bn', 'model.18.bn', ['model.18.bn', 'model.14.bn'], 'model.20.cv3.bn', 'model.21.bn', ['model.21.bn', 'model.10.bn'], 'model.23.cv3.bn'] 24 [17, 20, 23]
 24      [17, 20, 23]  1     21240  models.yolo.Detect                      [3, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [178, 295, 409]]
Model Summary: 391 layers, 12145411 parameters, 12145411 gradients, 42.3 GFLOPS

Transferred 504/506 items from runs/test/exp16/pruned_model.pt
Scaled weight_decay = 0.00036
Optimizer groups: 86 .bias, 86 conv.weight, 83 other
train: Scanning '/disk2/datasets/struct/labels/train.cache' images and labels... 26725 found, 1 missing, 9 empty, 5 corrupted: 10
val: Scanning '/disk2/datasets/struct/labels/val.cache' images and labels... 806 found, 0 missing, 0 empty, 0 corrupted: 100%|█|val: Scanning '/disk2/datasets/struct/labels/val.cache' images and labels... 806 found, 0 missing, 0 empty, 0 corrupted: 100%|█| 
Plotting labels... 
wandb: Network error (ConnectionError), entering retry loop.

请问在finetune pruned时候，这个wandb一直停在这里 该怎么解决？感谢

**Comment by Doris-miao** [comment]
> detect input : ['model.0.conv.bn', 'model.1.bn', 'model.2.cv3.bn', 'model.3.bn', 'model.4.cv3.bn', 'model.5.bn', 'model.6.cv3.bn', 'model.7.bn', 'model.8.cv2.bn', 'model.9.cv3.bn', 'model.10.bn', 'model.10.bn', ['model.10.bn', 'model.6.cv3.bn'], 'model.13.cv3.bn', 'model.14.bn', 'model.14.bn', ['model.14.bn', 'model.4.cv3.bn'], 'model.17.cv3.bn', 'model.18.bn', ['model.18.bn', 'model.14.bn'], 'model.20.cv3.bn', 'model.21.bn', ['model.21.bn', 'model.10.bn'], 'model.23.cv3.bn'] 24 [17, 20, 23] 24 [17, 20, 23] 1 21240 models.yolo.Detect [3, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [178, 295, 409]] Model Summary: 391 layers, 12145411 parameters, 12145411 gradients, 42.3 GFLOPS
> 
> Transferred 504/506 items from runs/test/exp16/pruned_model.pt Scaled weight_decay = 0.00036 Optimizer groups: 86 .bias, 86 conv.weight, 83 other train: Scanning '/disk2/datasets/struct/labels/train.cache' images and labels... 26725 found, 1 missing, 9 empty, 5 corrupted: 10 val: Scanning '/disk2/datasets/struct/labels/val.cache' images and labels... 806 found, 0 missing, 0 empty, 0 corrupted: 100%|█|val: Scanning '/disk2/datasets/struct/labels/val.cache' images and labels... 806 found, 0 missing, 0 empty, 0 corrupted: 100%|█| Plotting labels... wandb: Network error (ConnectionError), entering retry loop.
> 
> 请问在finetune pruned时候，这个wandb一直停在这里 该怎么解决？感谢
你可以把wandb卸载了，用tensorboad

**Comment by ningjianfeng** [comment]
是不是bacsize太大了
